src/engine/bank01.asm

```diff
@@ -1455,13 +1455,13 @@ PrintDuelResultStats:
 	sub [hl]
 .print_x_cards
 	call WriteTwoDigitNumberInTxSymbolFormat
-	ldtx hl, Text0080 ; CardsText
+	ldtx hl, CardsUnitText ; "枚"
```

Comment:
We usually wouldn't bother repeating this in a comment, but we may do so if it is particularly useful. Do you feel like this is helpful in this case?